playbooks/compute/nto/roles/configurecluster/tasks/main.yml

```diff
@@ -0,0 +1,47 @@
+---
+- name: Ensure python packages are installed
+  ansible.builtin.pip:
+    name:
+      - kubernetes >= 24.2.0
+      - PyYAML >= 3.11
+
+- name: Create performance Profile
```

Comment:
This part is not clear to me, or the yml playbook name is not clear. 
It sounds that the playbook create a directory, but the task name is create PP (also the var is artifacts_folder)...

Reply:
thanks the name was from a previous code